manuscript/the-meat.txt

```diff
@@ -434,7 +434,7 @@ Let’s add them.
 
 There’s nothing special going on here. `getYears` uses `_.groupBy` to group the data by year, then returns the keys of the array. We could make this more efficient, but it works well enough and is easy to read.
 
-`getFormatter` relies on the fact that d3.js scales have formatters. It creates a linear scale with a domain, and returns the `tickFormat()` function. Formatters don't work well without a defined domain.
+`getFormatter` relies on the fact that d3.js scales have formatters. It creates a linear scale with a domain and returns the `tickFormat()` function. Formatters don't work well without a defined domain.
```

Comment:
I think it complicates things. `that` separates the two parts into separate clauses. For once, I left it how it is.